indexing-hadoop/src/main/java/io/druid/indexer/HadoopDruidIndexerMapper.java

```diff
@@ -77,6 +80,7 @@ protected void map(
       }
       catch (Exception e) {
         if (config.isIgnoreInvalidRows()) {
+          log.warn(e, "Ignoring invalid row [%s] due to parsing error", value.toString());
```

Comment:
either ways, It is good to have ignoreInvalidRows. In some of our pipelines we are getting data from other systems which might be bad and we don't care about losing that portion. 
It will be too bad if this feature is not there and we have to have one full pass on the data just to filter the invalid and then ingest it to druid and that effectively increases our end-to-end ingestion time.

may be simple solution here is to just change the log.warn to log.debug so that it gets into logs only if user really wanted it in addition to the INVALID_ROW_COUNTER